js/main.js

```diff
@@ -129,4 +152,5 @@
   } // END initial
 
 // You are not expected to understand anything below this line...
-})(window || module && module.exports || this);
+});
+// (window || module && module.exports || this);
```

Comment:
This is what is screwing you up. You turned the _Immediately-Invoked Function Expression_ into just a _Function Expression_. Your code in `main.js` won't run unless the Function Expression is _invoked_...